src/assets/scripts/client/aircraft/AircraftInstanceModel.js

```diff
@@ -1314,8 +1314,12 @@ export default class AircraftInstanceModel {
         const isTimeToStartTurning = distanceToWaypoint < nm(calculateTurnInitiaionDistance(this, waypointPosition));
         const closeToBeingOverFix = distanceToWaypoint < PERFORMANCE.MAXIMUM_DISTANCE_TO_PASS_WAYPOINT_NM;
         const closeEnoughToFlyByFix = distanceToWaypoint < PERFORMANCE.MAXIMUM_DISTANCE_TO_FLY_BY_WAYPOINT_NM;
-        // TODO: abstract this logic to helper method
-        const shouldMoveToNextFix = closeToBeingOverFix || (closeEnoughToFlyByFix && isTimeToStartTurning);
+        const wantToFlyByFix = closeEnoughToFlyByFix && isTimeToStartTurning;
```

Comment:
:thinking: Now that you've got me thinking of it, that's not ideal either, because whether or not the aircraft "_should_" fly by the fix is actually determined by whether or not the fix is a fly-by fix (whether they should is essentially the question of the if statement).

What about `isTimeToFlyByFix`?